test/providers/quoters/WebhookQuoter.test.ts

```diff
@@ -65,6 +75,67 @@ describe('WebhookQuoter tests', () => {
     expect(response[0].toResponseJSON()).toEqual({ ...quote, quoteId: expect.any(String) });
   });
 
+  it('Only calls to eligible endpoints', async () => {
```

Comment:
nit: good to also put a comment here with what the threshold is for the breaker, and say which endpoints should be called / not called

Reply:
here we actually uses the `enabled` field in the config, the fade rate is there just for reference